source/extensions/omni.isaac.lab/test/assets/test_articulation.py

```diff
@@ -793,6 +793,185 @@ def test_apply_joint_command(self):
                         # are not properly tuned
                         assert not torch.allclose(articulation.data.joint_pos, joint_pos)
 
+    # def test_body_root_state_link_w_no_com_offset(self):
```

Comment:
Why is this commented out? If it isn't needed anymore, woudl suggest removing it. If it is failing, then if it is linked to a fix from PhysX we should make an issue.

Reply:
oh yeah its unneeded code. I consolidated it with the following function